src/screens/FeedScreen/ContentLink.js

```diff
@@ -37,8 +37,7 @@ const ContentLink = ({
   const sanitizeUrl = message.replace(/(https?:\/\/)?([^.\s]+)?[^.\s]+\.[^\s]+/gi, '').trim();
   const {hashtagAtComponent} = useContentFeed({navigation});
 
-  const {textHeight, heightTopic, handleTextHeight, handleTopicLayout} =
-    useCalculationContentLink();
+  const {textHeight, handleTextHeight, handleTopicLayout} = useCalculationContentLink();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `heightTopic` variable has been removed from the destructured object returned by `useCalculationContentLink()`. If `heightTopic` is no longer needed in this component, this is a good change. However, if it's still being used elsewhere in the component, this could cause issues. Please ensure that `heightTopic` is not used anywhere else in this component or update the code accordingly.
```diff
- const {textHeight, heightTopic, handleTextHeight, handleTopicLayout} = useCalculationContentLink();
+ const {textHeight, handleTextHeight, handleTopicLayout} = useCalculationContentLink();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->